applications/StructuralMechanicsApplication/tests/patch_test/updated_lagrangian/patch_test_3D_shear_hexa_parameters.json

```diff
@@ -10,10 +10,9 @@
         "echo_level"      : 0
```

Comment:
I just wonder if we need all of those files. We have one simple python_only file that includes everything, without json and parameters. Isn't that better instead of having a lot of mdpas and parameters to only have that?

to make it clear this is a question. Opinions are more than welcome

Reply:
Thats just the old format of the tests, isn't it?